Design-Docs/Image-Properties-Spec.md

```diff
@@ -211,3 +211,25 @@ A boolean property that is not present is considered to be `false`.
   pricing (but it might be provided by a contracted 3rd party, e.g. the OS vendor).
 
 
+##  Standardized Image  
+
+Images which are "default" in Glance of any SCS-Provider and are certified by SCS, should have following Features:
+
+*   Optional: Latest LTS Distribution of:
```

Comment:
So it's optional or mandatory to provide the latest LTS version? If it's optional why include this in the "standard"? In my mind there would be a mandatory section aswell if there is an optional section.

what happens if the next LTS version comes out, will the support be dropped for latest-1 version?